aiohttp/abc.py

```diff
@@ -146,3 +146,13 @@ def write_eof(self, chunk=b''):
     @abstractmethod
     def drain(self):
         """Flush the write buffer."""
+
+
+class AbstractAccessLogger(ABC):
+
+    def __init__(self, logger, log_format):
+        self.logger = logger
+
+    @abstractmethod
+    def log(self, request, response, time):
```

Comment:
Need to document which objects and types that method accepts.

Reply:
Sure, it will be documented